<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>salle</class>
 <widget class="QMainWindow" name="salle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font-family: century gothic;
font-size:14px

}
*{
background:#333;
}</string>
    </property>
    <property name="title">
     <string>Le Petit Prince</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>39</y>
       <width>701</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="title">
      <string>Gestion des salles</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>  Type du batiment :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Numero de salle :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Type de la salle :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
</string>
      </property>
      <property name="text">
       <string>Nom de la salle :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:15px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
}
QPushButton:hover
{
color:#333; 
border-radius:15px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>290</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:15px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
}
QPushButton:hover
{
color:#333; 
border-radius:15px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>salle ordinaire</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>salle de club</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>batiment crÃªche</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>batiment jardin d'enfant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>batiment garderie</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>391</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:Grey;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>240</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:15px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
}
QPushButton:hover
{
color:#333; 
border-radius:15px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>240</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:15px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
}
QPushButton:hover
{
color:#333; 
border-radius:15px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:15px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
}
QPushButton:hover
{
color:#333; 
border-radius:15px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:15px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
}
QPushButton:hover
{
color:#333; 
border-radius:15px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background:#49ebff ; 
border-radius:15px; 
color:#333; 
border: 2px solid gray ;
padding: 0.8px;
}
QPushButton:hover
{
color:#333; 
border-radius:15px;
background:#c7cf00;
border: 2px solid gray ;
padding: 0.8px;
}</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>salle</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>769</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
